src-java/base-topology/base-storm-topology/src/test/java/org/openkilda/wfm/topology/flowhs/service/FlowPathBuilderTest.java

```diff
@@ -68,7 +79,7 @@ public void setUp() {
     }
 
     @Test
-    public void shouldDetectSameSwitchPaths() {
+    public void detectSameSwitchPathsTest() {
```

Comment:
adding "Test" at the end is redundant. Same for the rest

Reply:
fixed